### ISSUE TITLE
Make SSH/Kubernetes token lifetimes from Teleport Discover more obvious

### ISSUE THREAD

**Issue by webvictim** [gravitational/teleport#19492]
**What would you like Teleport to do?**

When clicking the "Add server" button to start the Teleport Discover wizard, the 5 minute timer which counts down next to the install script link for adding a server makes it look like the token/script will only be valid for 5 minutes.

![Screenshot 2022-12-19 152840](https://user-images.githubusercontent.com/897821/208505382-e0332c0e-e2dd-4996-beb8-553bb0bb2650.jpg)

It's very unclear that the token and script link is actually still valid for 4 hours as it has always been:

```
gus@artemis:~$ sudo tctl tokens ls
Token                                                            Type     Labels Expiry Time (UTC)
---------------------------------------------------------------- -------- ------ ------------------------------
92b3561a...                                                      Node            19 Dec 22 23:28 UTC (3h58m37s)
```

This is confusing - both for people who were used to the old link which specified a 4 hour timer, and for people who might be trying to use the script to connect multiple servers within the timeframe.

Adding text near the link (like we used to have) saying "this link can be used as many times as needed within the next 4 hours" would be good.

The same thing should be done for the Kubernetes onboarding flow.

The database and application flows are fine, as they still have the old 4 hour text: 

![image](https://user-images.githubusercontent.com/897821/208506343-64310ab8-3c77-4445-b612-22273e474fca.png)

![image](https://user-images.githubusercontent.com/897821/208505962-29934bda-0ba0-407f-97d5-f0864e1d1cf6.png)

**What problem does this solve?**

Clarifies expectations of Teleport join links and makes them more useful than having to figure out how to add a join token manually in the docs and run someone through the convoluted experience of manually installing Teleport and adding a node.

**If a workaround exists, please include it.**

Teleport staff explain to people that the link is valid for longer.

**Comment by webvictim** [comment]
I believe this also affects Windows too.

**Comment by webvictim** [comment]
Fixed in #20829
